### DISCUSSION TITLE
tutorial to termux like

### DISCUSSION THREAD

**Discussion by MatejMagat305** [vlang/vsl#136] [Ideas]
@ulises-jeremias I know I m litle bit enoying, but I realy want discuss about whether tutorial is needed, I know how to do it, or I know add # flag -I/path/to/CL to source, but beginers will not know ..., if you not want to make it long, I will create repo where I will describe it and on this repo we can put "if you are on termux ", or extendable section, or else?
Btw can I do more example to vcl?

**Comment by ulises-jeremias** [comment]
Thanks for opening this discussion! I'll send tomorrow a small PR with a documentation so you can review it! I'll try to do it in a way that it works for Darwin and not Darwin OSs 👌🏻

**Comment by ulises-jeremias** [comment]
https://github.com/vlang/vsl/pull/139
